moveit_core/constraint_samplers/include/moveit/constraint_samplers/default_constraint_samplers.h

```diff
@@ -70,8 +69,25 @@ class JointConstraintSampler : public ConstraintSampler
    *
    */
   JointConstraintSampler(const planning_scene::PlanningSceneConstPtr& scene, const std::string& group_name)
-    : ConstraintSampler(scene, group_name)
-    , random_number_generator_(createSeededRNG("~joint_constraint_sampler_random_seed"))
+    : ConstraintSampler(scene, group_name), random_number_generator_(random_numbers::RandomNumberGenerator())
```

Comment:
I would use whatever the value is that `random_numbers` uses right now. Of course, it would be even better to just do away with that whole package but that's beyond the scope of this PR.

Reply:
I take back what I said about using the same seed as what `random_numbers` uses. It doesn't have a fixed seed... [https://github.com/ros-planning/random_numbers/blob/master/src/random_numbers.cpp](https://github.com/ros-planning/random_numbers/blob/master/src/random_numbers.cpp)

Given how many different parts of the code base might be using that package and therefore using a random number generator that gets seeded with different value every time the object is constructed, it might actually be a bad idea to change that behavior. :thinking: